databricks/koalas/indexes.py

```diff
@@ -2973,6 +2973,30 @@ def item(self):
         """
         return self._kdf.head(2)._to_internal_pandas().index.item()
 
+    def equal_levels(self, other):
+        """
+        Return True if the levels of both MultiIndex objects are the same
+
+        Examples
+        --------
+        >>> kmidx1 = ks.MultiIndex.from_tuples([("a", "x"), ("b", "y"), ("c", "z")])
+        >>> kmidx2 = ks.MultiIndex.from_tuples([("b", "y"), ("a", "x"), ("c", "z")])
+        >>> kmidx1.equal_levels(kmidx2)
+        True
+        """
+        nlevels = self.nlevels
+        if nlevels != other.nlevels:
+            return False
+        self = self.sort_values()
+        other = other.sort_values()
```

Comment:
This method can be quite expensive because we have to sort all the values.

I think maybe we can have an option that support this function for the only exactly same `MultiIndex`.

like

```python
>>> kmidx1 = ks.MultiIndex.from_tuples([("a", "x"), ("b", "y"), ("c", "z")])
>>> kmidx2 = ks.MultiIndex.from_tuples([("b", "y"), ("a", "x"), ("c", "z")])
>>> kmidx1.equal_levels(kmidx2)
False

>>> kmidx1 = ks.MultiIndex.from_tuples([("a", "x"), ("b", "y"), ("c", "z")])
>>> kmidx2 = ks.MultiIndex.from_tuples([("a", "x"), ("b", "y"), ("c", "z")])
>>> kmidx1.equal_levels(kmidx2)
True
```